pySHiELD/update/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .fv_update_phys import ApplyPhysicsToDycore
```

Comment:
@oelbert same comment than for `pyFV3` on small docstring

Reply:
For these it's pretty straightforward so definitely think that's a good idea here.